polyfill/lib/absolute.mjs

```diff
@@ -16,39 +16,33 @@ import {
   NANOSECONDS
 } from './slots.mjs';
 import { absolute as STRING } from './regex.mjs';
+import bigInt from 'big-integer';
```

Comment:
why this one and not jsbi?

Reply:
Can you elaborate how jsbi would be a better pick in your opinion in this particular case?

Reply:
jsbi is maintained and created by TC39 delegates, and as such I'd expect it to have better spec compliance.

Reply:
Why not jsbi?

Simple, frequenc of use. jsbi has roughly 2.5e4 weekly downloads, big-integer 1.6e6 so we are talking 2 orders of magnitude. Both seem to basically do the same things and have the same limitations. So why would I use jsbi?

Reply:
That's likely a function of age; big-integer long predates it and was created before there was a proposal, jsbi was created *explicitly* for the language proposal.

Reply:
Which is entirely irrelevant since it’s used just for internal maths and never exposed out.

Reply:
It's checking if a user-provided argument is an instance of the polyfill, so it's absolutely both relevant and exposed, since if i pass in an instance of this library i get different behavior than with other objects.